### ISSUE TITLE
Failing partway through | sandboxed

### ISSUE THREAD

**Issue by scarnecchia** [mapbox/tippecanoe#38]
These may be two separate issues, but tippecanoe is getting 15-20% of the way through a task before failing without an error message.

```
Digital-Aurora-MacBook-Pro:amtrak scarndp$ cat USRailNetwork.geojson | tippecanoe -o USRailNetwork2.mbtiles -z4 -Z4 -d6 -g2 -pf -pl -f
using layer 0 name USRailNetwork2
Sorting 172983 features
172983 features, 20913187 bytes of geometry, 69280157 bytes of metadata
  19.9%  0/0/0  
Digital-Aurora-MacBook-Pro:amtrak scarndp$
```

Over on the console, it appears to begetting sandboxed:
```
mdworker(5162) deny file-write-data /Users/scarndp/Github/amtrak/USRailNetwork2.mbtiles (import fstype:hfs fsflag:480D000 flags:240000005E diag:0 isXCode:0 uti:com.mapbox.mbtiles plugin:/TileMill.app/Contents/Library/Spotlight/MBTiles.mdimporter - find suspect file using: sudo mdutil -t 25152825)
```

Model Identifier: MacBookPro11,1
Intel(R) Core(TM) i5-4258U CPU @ 2.40GHz
2 GB RAM
Yosemite

**Comment by ericfischer** [comment]
Thanks for the report @scarnecchia. I've never seen that error myself. Is the file you're trying to encode public so I could try it out myself?

**Comment by scarnecchia** [comment]
@ericfischer It's looking to be user error.

I'd created a geojson file from the Railway Network shapefile at the [National Transportation Atlas Database](http://apps.bts.gov/publications/national_transportation_atlas_database/2012/) using QGIS, and when I exported it, accidentally saved it as a different projection. When I corrected this, everything worked. The Spotlight issue that sandbox is killing still appears, but doesn't appear to be effecting output.
